redis/balanceCache.ts

```diff
@@ -0,0 +1,56 @@
+import { Prisma } from '@prisma/client'
+import { AddressPaymentInfo, generateAddressPaymentInfo } from 'services/addressService'
+import { TransactionWithAddressAndPrices } from 'services/transactionService'
+import { redis } from './clientInstance'
+import { Payment } from './types'
+
+// ADDRESS:balance
```

Comment:
?

Reply:
This is actually the real file, the other one I accidentally put in the root dir, no idea how. I am deleting that other file so the thread will disappear, pasting here the answer I had left there:

---
![image](https://github.com/PayButton/paybutton-server/assets/21281174/08cc7b9c-545a-4919-959a-89e240376f8f)